dask_geopandas/core.py

```diff
@@ -336,6 +337,32 @@ def hilbert_distance(self, p=15):
 
         return distances
 
+    def geohash(self, precision=15):
+
+        """
+        A function that encodes geometry mid points using Geohash for a given precision
```

Comment:
Can you define in the docstring what a "mid point" is?

Reply:
updated in b8f6e38

Reply:
@tastatham it looks like this got updated for the internal `_geohash()` but not the top level `geohash()` here.

Reply:
updated in df72e775c33f4ec98331048b3715ae8dcbbcfdea